var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -164,10 +164,36 @@ def install(self, spec, prefix):
 
         install_tree('lib', prefix.lib)
         install_tree('include', prefix.include)
+
+        # Create separate libexec directory tree for libraries and
+        # headers that includes a "fake MPI" header needed for
+        # sequential MUMPS builds (i.e., without MPI) when linking
+        # MUMPS to some libraries (e.g., IPOPT). This "fake MPI"
+        # header will shadow headers in MPI implementations (e.g.,
+        # MPICH, OpenMPI), so there should two separate include
+        # directories: (1) without the "fake MPI" header
```

Comment:
can you use MUMPS build with `+mpi` in sequential mode (i don't know)? If not, perhaps all these extra steps should be done for `~mpi` and then one wouldn't need an extra `libexec/include`?